packages/artifacts/README.md

```diff
@@ -0,0 +1,101 @@
+# @aragon/osx-artifacts
+
+## Installation
+
+    ```bash
+    npm install @aragon/osx-artifacts
+    ## or
+    yarn add @aragon/osx-artifacts
+    ```
+
+## Usage
+
+### 1.3.1 and previous versions
+
+Versions `1.3.1`and previous only export the json artifacts. To use them just import the artifacts and use them as needed.
+
+```ts
+import {DAOFactory} from '@aragon/osx-artifacts';
+
+console.log(DAOFactory.abi);
+console.log(DAOFactory.bytecode);
+```
+
+### 1.4.0, versions with `-viem` suffix and future versions
```

Comment:
put this at the top of the readme as it will be default behaviour

Reply:
[198b9cf](https://github.com/aragon/osx/pull/592/commits/198b9cf1919e57ee147bf8611e95dc4b5124f903)